app/views/artists/registrations/new.html.erb

```diff
@@ -1,38 +1,46 @@
 <div class="loginPage">
-  <div class="container">
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+
+  <div class="container signup part1">
     <h1>Sign up</h1>
+    <%= devise_error_messages! %>
 
-    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
-      <%= devise_error_messages! %>
+    <div class="mb3">
+      <%= f.label :email %>
+      <%= f.email_field :email, class: "textinput w-100", autofocus: true, autocomplete: "email" %>
+      <p class="texthelp"> artists.sfai.edu email required </p>
+    </div>
 
-      <div class="mb3">
-        <%= f.label :email %>
-        <%= f.email_field :email, class: "textinput w-100", autofocus: true, autocomplete: "email" %>
-        <p class="texthelp"> artists.sfai.edu email required </p>
-      </div>
+    <div class="mb3">
+        <%= f.label :password %>
+        <%= f.password_field :password, class: "textinput w-100", autocomplete: "new-password" %>
+        <% if @minimum_password_length %>
+          <p class="texthelp"> <%= @minimum_password_length %> characters minimum </p>
+        <% end %>
+    </div>
 
-      <div class="mb3">
-          <%= f.label :password %>
-          <%= f.password_field :password, class: "textinput w-100", autocomplete: "new-password" %>
-          <% if @minimum_password_length %>
-            <p class="texthelp"> <%= @minimum_password_length %> characters minimum </p>
-          <% end %>
-      </div>
+    <div class="mb4">
+      <%= f.label :password_confirmation, class: "mb1" %>
+      <%= f.password_field :password_confirmation, class: "textinput w-100", autocomplete: "new-password" %>
+    </div>
 
-      <div class="mb3">
-        <%= f.label :password_confirmation, class: "mb1" %>
-        <%= f.password_field :password_confirmation, class: "textinput w-100", autocomplete: "new-password" %>
-      </div>
+    <div class="flex justify-between">
+      <%= render "artists/shared/links" %>
+      <button type="button" onclick="nextPage()" class="button-primary bg-mustard w4">Next</button>
+    </div>
+  </div>
 
-      <div class="mb4">
+  <div class="container signup part2 dn">
+
+    <%= render "artists/shared/terms" %>
+
+    <div class="flex justify-between items-baseline">
+      <div>
         <%= f.check_box :terms_and_conditions %>
         <%= f.label :terms_and_conditions, "I agree to SFAI's Terms and Conditions" %>
       </div>
-
-      <div class="flex justify-between">
-        <%= render "artists/shared/links" %>
-        <%= f.submit "Sign up", class: "button-primary bg-mustard w4" %>
-      </div>
-    <% end %>
+      <%= f.submit "Sign up", class: "button-primary bg-mustard w4" %>
```

Comment:
Can we do inline validation for the checkbox? Like if the user doesn't check the box, don't let them submit the form?